README.md

```diff
@@ -3,15 +3,25 @@
 Welcome to the Personal Travel Assistant project! This application is designed to assist you with your travel needs and make your journey more enjoyable.
 
 ## How to Run
+<ol>
+    <li>
+        <p>
+            Before you begin, make sure you have <a href="https://nodejs.org/en/download/current">Node.js</a> installed on your system.
+        </p>
+    </li>
+    <li>
+        <p>After that, go into the travel-assistant folder</p>
```

Comment:
This is not necessary

Reply:
it's necessary, as the project folder is not the root of the git repo